.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,16 @@
+name: Unit_test
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+    - name: Install dependencies
+      run: bundle install
```

Comment:
Better use
```
      run: |
        gem install bundler
        bundle install --jobs 4 --retry 3
```

This allow setup gems in parallel (great speedup) and retry on network failures)

Like here:
https://github.com/onlyoffice-testing-robot/onlyoffice_logger_helper/blob/master/.github/workflows/ruby.yml#L18